### PR TITLE
release: Make release-source more verbose

### PR DESCRIPTION
**Pull request by stefwalter**
We need to be able to better diagnose build failures.

### CODE REVIEW

**Comment by stefwalter** [comment]
This needs testing during a release run.

File: release/release-source [hunk 1]
```diff
@@ -248,6 +250,8 @@ shift $(expr $OPTIND - 1)
 
 if [ $VERBOSE -eq 1 ]; then
     set -x
+elif [ $QUIET -eq 0 ]; then
+    MAKE="make --silent"
```
Comment (martinpitt) [inline release/release-source]: This looks the wrong way around -- should't we use `--silent` if `$QUIET` is true, i. e. `-eq 1`, not 0?

File: release/release-source [hunk 2]
```diff
@@ -140,7 +142,7 @@ prepare()
                 trace "Creating first tarball"
 
                 autogen_or_configure $repodir
-                make -C $repodir/_build --silent dist
+                $(MAKE) -C $repodir/_build dist
```
Comment (martinpitt) [inline release/release-source]: and this needs to be `$MAKE`, this isn't a Makefile (confusingly ☺).

**Comment by martinpitt** [comment]
136 was released with this patch in our container, and the log is verbose: https://fedorapeople.org/groups/cockpit/logs/release-136/log

So I consider this sufficiently tested.

**Pull request merged**